pkg/mounter/ossfs.go

```diff
@@ -355,7 +357,11 @@ func CleanupOssfsCredentialSecret(ctx context.Context, clientset kubernetes.Inte
 }
 
 func getRoleSessionName(volumeId, target string) string {
-	return fmt.Sprintf("ossfs.%s.%s", volumeId, computeMountPathHash(target))
+	name := fmt.Sprintf("ossfs.%s.%s", volumeId, computeMountPathHash(target))
```

Comment:
So the whole role name length depends on the length of volumeId? btw i don't think this is a good idea that we have two pattern names.

Reply:
I also think truncating is better.

Reply:
```suggestion
	name := fmt.Sprintf("ossfs.%s.%s", computeMountPathHash(target), volumeId)
```
OK. But how about this? So that we always have a complete hash at least. The hash maybe used as label selector for diagnose.